src/dotnet/Chat.UI.Blazor/Components/ChatMenu/chat-menu.css

```diff
@@ -1,19 +1,35 @@
-.navbar-item .chat-menu-btn {
+.visible-menu {
```

Comment:
For me ok. But it seems require some refactoring in a separate task.
Styles are quite complex and spreaded along many css files.
Look for 
.navbar-item:hover .pin-toggle.off {
    @apply visible;
}
in navbar.css
and 
.chats-tab .btn-round,
.contacts-tab .btn-round,
.listening-chats .btn-round {
    @apply w-6 h-6;
}
in ChatUI component.css
I suppose that introducing chat-list and chat-list-item css classes may help.